### PR TITLE
Add jobs for HA testing

### PR DESCRIPTION
**Pull request by HartS**

### CODE REVIEW

File: helm-deploy-test/helm-deploy-test.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ resources:
     pool: ((kube-pool-pool))
 
 jobs:
-- name: helm-deploy-test-openSUSE
+- name: helm-deploy-sa-test-openSUSE
```
Comment (prabalsharma) [inline helm-deploy-test/helm-deploy-test.yml]: did u mean "ha", rather than "sa-"

File: helm-deploy-test/helm-deploy-test.yml [hunk 2]
```diff
@@ -82,6 +82,103 @@ jobs:
   - put: pool.kube-hosts
     params: {release: pool.kube-hosts}
 
+- name: helm-deploy-sa-test-SLES
```
Comment (prabalsharma) [inline helm-deploy-test/helm-deploy-test.yml]: -ha-

File: helm-deploy-test/helm-deploy-test.yml (hunk 1 above)
Comment (prabalsharma) [reply helm-deploy-test/helm-deploy-test.yml]: I guess we should either remove non-ha opensuse or ha-opensuse jobs, we should not use 2 resources(caasp) to test opensue. 1 out of two should be enough. I guess we should keep HA and remove nonHA opensuse job

Comment (HartS) [reply helm-deploy-test/helm-deploy-test.yml]: sa if for single/standard availability

**Pull request merged**